client/components/jetpack/card/jetpack-product-card-alt-2/features-product-slide-out.tsx

```diff
@@ -77,24 +78,38 @@ const FeaturesProductSlideOut: FunctionComponent< Props > = ( {
 			slug === productSlug ||
 			JETPACK_PRODUCTS_LIST[ slug as JetpackProductSlug ]?.type === productSlug
 	);
+
 	const isItemPlanFeature = !! ( sitePlan && planHasFeature( sitePlan.product_slug, productSlug ) );
 	const isOwned = isItemPlanFeature || isItemPurchased;
+	const isUpgradeableToYearly =
+		( isOwned &&
+			productBillingTerm === TERM_ANNUALLY &&
+			monthlyProductSlug &&
+			purchases.some( ( { productSlug } ) => productSlug === monthlyProductSlug ) ) ||
+		false;
 	const purchase = isItemPlanFeature
 		? getPurchaseByProductSlug( purchases, sitePlan?.product_slug || '' )
 		: getPurchaseByProductSlug( purchases, productSlug ) ||
 		  ( monthlyProductSlug
 				? getPurchaseByProductSlug( purchases, monthlyProductSlug )
 				: undefined );
-	const slideOutButtonLabel = isOwned
-		? translate( 'Manage Subscription' )
-		: translate( 'Get {{name/}} %(price)s', {
-				args: {
-					price: formattedPrice,
-				},
-				components: {
-					name: <>{ shortName }</>,
-				},
-		  } );
+
+	const slideOutButtonLabel = useCallback( () => {
```

Comment:
Nitpicking here (non-blocking comment). For cases like this, I think `useMemo` is more adequate. We don't really need a function, we only need a computed value which is the purpose of `useMemo`. For me, `useCallback` works better when we want to pass down a function that we don't want to re-create on each render cycle.

Reply:
Yes definitely, Good call. Thanks! 👍